### PR TITLE
PR: Algo shell sort in python

### PR DESCRIPTION
**Pull request by Vybhavigithub**
Shellsort algorithm in python is placed in Algorithm-python folder.

### CODE REVIEW

File: 2-algorithms/3-python/Shellsort.MD [hunk 1]
```diff
@@ -0,0 +1,43 @@
+"""
```
Comment (Abishethvarman) [inline 2-algorithms/3-python/Shellsort.MD:1]: Add h1 heading

**Review by Abishethvarman — CHANGES_REQUESTED** [review]

**Comment by Vybhavigithub** [comment]
@Abishethvarman please do review

File: 2-algorithms/3-python/Shellsort.MD (hunk 1 above)
Comment (Abishethvarman) [reply 2-algorithms/3-python/Shellsort.MD:3]: your content seems good, but we need the definition, simple explantion and the code level explanation. please refer this MD file and make changes accordingly https://github.com/Abishethvarman/programming-resources/blob/main/2-algorithms/1-java/merge-sort.MD

**Review by Abishethvarman — CHANGES_REQUESTED** [review]

**Comment by Vybhavigithub** [comment]
@Abishethvarman I've made changes accordingly . Please do check.

File: 2-algorithms/3-python/Shellsort.py
Comment (Abishethvarman) [inline 2-algorithms/3-python/Shellsort.py:1]: no py file needed. one single md file is enough

**Review by Abishethvarman — CHANGES_REQUESTED** [review]

File: 2-algorithms/3-python/Shellsort.py
Comment (Vybhavigithub) [reply 2-algorithms/3-python/Shellsort.py:1]: okay

**Review by Abishethvarman — APPROVED** [review]
Approved to merge

**Comment by Abishethvarman** [comment]
@Thevapriyan07 approve this PR

**Review by Thevapriyan07 — APPROVED** [review]
Approve to merge

**Pull request merged**